packages/apps/docs/src/components/Markdown/Code/Code.tsx

```diff
@@ -1,15 +1,40 @@
-import { StyledCode, StyledInlineCode } from './styles';
+import { code, codeLine, inlineCode } from './style.css';
 
-import React, { FC, ReactNode } from 'react';
+import { useTheme } from '@/hooks';
+import React, { FC, ReactNode, useEffect } from 'react';
 
 interface IProp {
   children: ReactNode;
 }
 
 export const Code: FC<IProp> = ({ children, ...props }) => {
+  const { theme } = useTheme();
+
+  const [themedProps, setThemedProps] = React.useState(props);
+
+  useEffect(() => {
+    setThemedProps({
+      ...props,
+      'data-active-theme': theme,
+    });
+  }, [props, theme]);
+
```

Comment:
what is the purpose of this theme stuff?

Reply:
Carried over from the original implementation:
https://github.com/kadena-community/kadena.js/blob/277bd6904a9a229a84c42efd6cd389c56d20011b/packages/apps/docs/src/components/Markdown/Code/styles.ts#L87

For some reason both a light and a dark themed version of the code block appear, and one must be hidden.

Reply:
Managed to get it done with pure CSS